fern/pages/sdks/features/discriminated-unions.mdx

```diff
@@ -1,33 +1,77 @@
-  ### Discriminated Unions
-  The SDKs natively support discriminated unions. 
+### Discriminated Unions
+The SDKs natively support discriminated unions.
```

Comment:
maybe we can mention `oneOf` and `anyOf` in OpenAPI and show an example of how to define? i think youll want to use an accordion to show `Fern Definition` and `OpenAPI spec`